.github/workflows/api-dotnetcore.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         run: dotnet restore
         working-directory: ${{env.working-directory}}
       - name: Install coverlet for code coverage
-        run: dotnet tool install -g coverlet.console
+        run: dotnet tool install -g coverlet.console --ignore-failed-sources
```

Comment:
Is this to fix the coverlet GitHub action issue?  The error makes me wonder if we should be using a specific version, or a release version at least.

Reply:
It was an attempt, no luck with it though. And yeah me too, I am going to try a few other things here.